.github/workflows/ci.yml

```diff
@@ -98,7 +131,7 @@ jobs:
         run: php artisan route:list
 
       - name: Execute tests (Unit and Feature tests) via PHPUnit
-        run: php artisan test --parallel --configuration phpunit-ci.xml --testsuite=Feature,Unit
```

Comment:
Did we lose the ability to run tests in `parallel` with this configuration?

Reply:
For now, yes, I am sure we can do it. But it gave me enough trouble to warrant not handling it in this POC.